docs/shadow.md

```diff
@@ -51,6 +53,32 @@ To locate the Shadow DOM in above HTML,
 I.click({custom: '.primary-button'});
 ```
 
+If you want to provide another type of locator, give it another name.
+
+```js
+// in codecept.conf.js
+const { locatorStrategy } = require('query-selector-shadow-dom/plugins/webdriverio');
+
+const myStrat = (selector) => {
+  return document.querySelectorAll(selector)
+}
+
+// under WebDriver Helpers Configuration
+WebDriver: {
+  ...
```

Comment:
this was the main change instead of hardcoding to one strategy, make use of that locators.js already supports undefined custom locators, with `isCustom` and we can build  up as many custom locators as we want